test_util/aws.py

```diff
@@ -99,6 +97,18 @@ def create_stack(self, name, template_url, parameters, deploy_timeout=60):
             # so that we may pass parameters directly from yaml (which parses numbers as non-strings)
             Parameters=param_dict_to_aws_format(parameters))
 
+    def create_stack_from_string(self, name, template, parameters, deploy_timeout=60):
```

Comment:
> You could literally pass the args straight through with **kwargs, and then the caller of this method has to choose between providing the TemplateURL arg or the TemplateBody arg. That kind of defeats the purpose of the method.

Not sure how -- that's the API as provided by CloudFormation. One function that takes either `TemplateBody` or `TemplateURL`.

> Alternatively, you can't just default both TemplateBody and TemplateURL as None and then expect this to work when you fill one of them in.

I'm doubtful, since that's how boto's `create_stack()` is implemented: https://github.com/boto/boto/blob/abb38474ee5124bb571da0c42be67cd27c47094f/boto/cloudformation/connection.py#L306